docs/install.md

```diff
@@ -0,0 +1,116 @@
+# Installing kpack
+
+* [Pre-requisites](#pre-requisites)
+* [Installing kpack](#installing-kpack)
+
+
+## Pre-requisites
+
+1. A Kubernetes cluster version 1.12 or later
+1. [kubectl cli](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+1. Cluster-admin permissions for the current user.
+1. Accessible Docker V2 Registry
+
+## Installing-kpack
+
+1. Download the most recent [github release](https://github.com/pivotal/kpack/releases).
+
+   ```bash
+   kubectl apply  --filename release.yaml
+   ```
+1. Ensure that the kpack controller has a status of `Running` using  `kubectl get`.   
+
+   ```bash
+   kubectl get pods --namespace kpack --wait
+   ```
+
+1. Create a [ClusterBuilder](builders.md) resource. A ClusterBuilder is a reference to a [Cloud Native Buildpacks builder image](https://buildpacks.io/docs/using-pack/working-with-builders/). 
+The Builder image contains buildpacks that will be used to build images with kpack. We recommend starting with the [cloudfoundry/cnb:bionic](https://hub.docker.com/r/cloudfoundry/cnb) image which has support for java, node and go.         
+
+```yaml
+apiVersion: build.pivotal.io/v1alpha1
+kind: ClusterBuilder
+metadata:
+  name: default-builder
+spec:
+  image: cloudfoundry/cnb:bionic
+```
+
+Apply the ClusterBuilder yaml to the cluster
+
+```bash
+kubectl apply -f <name-of-cluster-builder-file.yaml>
```

Comment:
This whole section on installing the clusterbuilder is not needed. The 0.0.4 release installs it already.

Reply:
Yeah, we are going to remove that from the release 😞

Reply:
Aw, OK